src/main/java/io/dataspaceconnector/model/configuration/Configuration.java

```diff
@@ -85,6 +86,13 @@ public class Configuration extends NamedEntity {
      */
     private String version;
 
+    /**
+     * The public key.
+     */
+    @Lob
+    @Column(columnDefinition = "TEXT")
+    private byte[] publicKey;
```

Comment:
Why is it defined as byte array?

Reply:
1. To long value for a simple string
2. most important: IAIS Java Artifact requires it to be byte[] in its builder-classes, therefore to have consistency in data management

Reply:
https://github.com/International-Data-Spaces-Association/Java-Representation-of-IDS-Information-Model/blob/7d354b04f4c0dd8888dfba8ff1712d94574d080d/de/fraunhofer/iais/eis/PublicKeyImpl.java#L56